### PR TITLE
[Minor] Fix Variable initialization

### PR DESCRIPTION
**Pull request by satybald**
Combine variable declartion and intiaztiona together.

### CODE REVIEW

**Comment by satybald** [comment]
I'm wondering what do the team think about enabling style checks plugin? I noticed that KSQL inherits from confluent common, where the code style is [defined](https://github.com/confluentinc/common/blob/master/pom.xml#L299). But code style issues de not fail the build.

**Comment by satybald** [comment]
For some reason, commit fails and I cannot see why. Would it be possible to share the result of the build? @hjafarpour @miguno @dguy @bluemonk3y

**Comment by bluemonk3y** [comment]
retest this pls

**Comment by dguy** [comment]
Checkstyle is, or at least was enabled and should fail the build. However, i'm not sure which checkstyle rules it is using at this stage as there is clearly code that should fail the checks. Will need to investigate further

File: ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java [hunk 1]
```diff
@@ -364,13 +364,12 @@ private Schema buildAggregateSchema(final Schema schema,
...
     for (int aggFunctionVarSuffix = 0;
          aggFunctionVarSuffix < getFunctionList().size(); aggFunctionVarSuffix++) {
-      Schema fieldSchema;
       String udafName = getFunctionList().get(aggFunctionVarSuffix).getName()
           .getSuffix();
       KsqlAggregateFunction aggregateFunction = functionRegistry.getAggregateFunction(udafName,
           getFunctionList()
               .get(aggFunctionVarSuffix).getArguments(), schema);
-      fieldSchema = aggregateFunction.getReturnType();
+      Schema fieldSchema = aggregateFunction.getReturnType();
```
Comment (dguy) [inline ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java]: Can probably take this one step further and just inline it. There is no reason for having the variable

**Comment by dguy** [comment]
retest this please

**Comment by satybald** [comment]
my pleasure @dguy. Wondering when this patch can be merged?

**Comment by bluemonk3y** [comment]
jenkins ran out of disk

**Comment by bluemonk3y** [comment]
retest this please

**Comment by satybald** [comment]
@bluemonk3y thanks for the information. What're your thoughts about having a public Travis CI build system? You can still have a private confluent Jenkins and ran all internal integration tests there.

**Comment by dguy** [comment]
retest this please

**Comment by satybald** [comment]
@dguy @bluemonk3y @hjafarpour  locally it compiles without any problem. However, the branch contains flaky integration tests that fails the build. 
compilation

**Comment by dguy** [comment]
It is failing because there is no space left on the disk jenkins is using. Unfortunately we can't merge until we get a clean build. Sorry for the wait @satybald

**Comment by satybald** [comment]
@dguy I tested it locally. I noticed that integration tests in serde module do not pass.

**Comment by bluemonk3y** [comment]
retest this please

**Comment by dguy** [comment]
Build finally passed. Thanks for your patience @satybald

**Comment by satybald** [comment]
yea :tada:

**Pull request merged**